.do/values.yaml

```diff
@@ -83,6 +83,11 @@ global:
     service:
       port: <--- codacy-db-port --->
 
+  metrics:
+    grafana:
+      enabled: true
+      label: "grafana_dashboard"
```

Comment:
Required to decorate grafana_dashboard configmaps, otherwise grafana doesn't pick them up